colossalai/fx/tracer/bias_addition_patch/patched_bias_addition_function/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .addmm import *
+from .bias_addition_function import *
```

Comment:
I suggest avoiding using *.
point out the function names, and use __all__ = ['XXX'] to specific the function you would like to expose to users.

Reply:
sure, I will fix it